Doc/library/csv.rst

```diff
@@ -347,7 +347,7 @@ The :mod:`csv` module defines the following constants:
    ``None``.  This is similar to :data:`QUOTE_ALL`, except that if a
    field value is ``None`` an empty (unquoted) string is written.
 
-   Instructs :class:`reader` objects to interpret an empty (unquoted) field as None and
+   Instructs :class:`reader` objects to interpret an empty (unquoted) string as ``None`` and
```

Comment:
Why change this?

Reply:
Fixed formatting for `None` and changed "field" to "string" as "unquoted string" make more sense. This was the only place where "field" is used, in all other similar contexts "string" is used.

Reply:
I don’t agree that this should be changed. I think you misunderstand the meaning of this fragment.

Reply:
https://github.com/python/cpython/blob/31aaff5a178e1daa073ab9fb3ceeba2220bbe2aa/Modules/_csv.c#L641-L656

In the implementation both constants define the same behavior for an empty unquoted string - such strings are converted to `None`. But the docs differ.

> Instructs reader objects to interpret an empty (unquoted) **field** as `None` and to otherwise behave as QUOTE_ALL.

> Instructs reader objects to interpret an empty (unquoted) **string** as `None` and to otherwise behave as QUOTE_NONNUMERIC.

Since there is no difference in the behavior for the case, I see no reason to use different lexica. I selected "string" over "field" as phrase "empty (unquoted) string" is more common in this piece of docs.

Reply:
I noticed that typo with `None` got fixed in 3.12 docs, so I removed that commit from this PR. The docs still say `empty (unquoted) field`, but it can be addressed in other PR later.

Given that the topic of this conversation is no longer here, I am resolving it.